test/index.js

```diff
@@ -94,7 +94,7 @@ describe('Tweedentity Api integration tests', function () {
 
 		it('should validate the params - bad twitter id too long', function () {
 			return request(app)
-				.get('/tweet/12345678901234567891/0x0123456789012345678901234567890123456789')
+				.get('/tweet/123456789012345678901/0x0123456789012345678901234567890123456789')
```

Comment:
The two checks were in conflict because the re accepted a tweet 20 chars long, while the length check wanted the tweetId being < 20. Leaving only the first one, now a string between 18 and 20 is accepted. This is fine.